fontbe/src/metrics_and_limits.rs

```diff
@@ -44,7 +44,11 @@ impl GlyphLimits {
         let bbox = glyph.bbox();
         let left_side_bearing = bbox.x_min;
         // aw - (lsb + xMax - xMin) ... but if lsb == xMin then just advance - xMax?
-        let right_side_bearing = advance.saturating_sub(bbox.x_max.try_into().unwrap());
+        let right_side_bearing = if bbox.x_max > 0 {
+            advance.saturating_sub(bbox.x_max.try_into().unwrap())
+        } else {
+            0
```

Comment:
why can't the bbox.x_max and thus the right side bearing be negative? hhea.minRightSideBearing is an FWORD and as such signed

this is how ufo2ft computes the hhea min_{left,right}_side_bearings
https://github.com/googlefonts/ufo2ft/blob/0d2688cd847d003b41104534d16973f72ef26c40/Lib/ufo2ft/outlineCompiler.py#L778-L797

Reply:
GlyphLimits::min_right_side_bearing is currently typed `Option<u16>` but it should be `Option<i16>` instead (i.e. FWORD)